docs/modules/mysql.md

```diff
@@ -9,9 +9,9 @@ Open Service Broker for Azure contains three Azure Database for MySQL services.
 |--------------|-------------|
 | `azure-mysql` | Provision both an Azure Database for MySQL Database Management System (DBMS) and a database. |
 | `azure-mysql-dbms-only` | Provision only an Azure Database for MySQL DBMS. This can be used to provision multiple databases at a later time. |
-| `azure-mysql-db-only` | Provision a new database only upon a previously provisioned DBMS. |
+| `azure-mysql-database-only` | Provision a new database only upon a previously provisioned DBMS. |
 
-The `azure-mysql` service allows you to provision both a DBMS and a database. This service is ready to use upon successful provisioning. You can not provision additional databases onto an instance provisioned through this service. The `azure-mysql-dbms-only` and `azure-mysql-db-only` services, on the other hand, can be combined to prprovisionovison multiple databases on a single DBMS.  For more information on each service, refer to the descriptions below.
+The `azure-mysql` service allows you to provision both a DBMS and a database. This service is ready to use upon successful provisioning. You can not provision additional databases onto an instance provisioned through this service. The `azure-mysql-dbms-only` and `azure-mysql-database-only` services, on the other hand, can be combined to provision multiple databases on a single DBMS.  For more information on each service, refer to the descriptions below.
```

Comment:
"can not" > can't (or cannot)

Reply:
s/This service is ready to use upon successful provisioning/When the provision operation is successful, the database will be ready to use